dashboard/src/components/ConfirmDialog/ConfirmDialog.v2.tsx

```diff
@@ -22,17 +24,19 @@ function ConfirmDialog({
   confirmationText,
   onConfirm,
   closeModal,
+  error,
 }: IConfirmDialogProps) {
   return (
     <Modal showModal={modalIsOpen} onModalClose={closeModal}>
+      {error && <Alert theme="danger">Found error: {error.message}</Alert>}
       {loading === true ? (
         <div className="confirmation-modal">
-          <div>Loading, please wait</div>
+          <span>Loading, please wait</span>
           <LoadingWrapper loaded={false} />
         </div>
       ) : (
         <div className="confirmation-modal">
-          <div>{confirmationText || "Are you sure you want to delete this?"}</div>
```

Comment:
I wouldn't use the word "this" in these (not your change, I know). Ah - I see, it's just the generic text if `confirmationText` isn't provided. Perhaps we should just make `confirmationText` required (not necessary here).